cmd/portmirror/portmirror.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"os/signal"
+	"sync"
+	"time"
+
+	"github.com/sirupsen/logrus"
```

Comment:
I could switch to klog, tho I would have to write a wrapper because the goflow library expects a implementation of its own interface. Sadly klog doesnt have a struct that implements that already. goflow itself uses logrus, which is why I just copied it